torchsynth/module.py

```diff
@@ -654,27 +729,46 @@ class TorchSynth(nn.Module):
     buffer_size (int): number of samples expected at output of child modules
     """
 
-    def __init__(self, sample_rate: int = SAMPLE_RATE, buffer_size: int = BUFFER_SIZE):
+    def __init__(
+        self,
+        synthglobals: TorchSynthGlobals,
+    ):
         super().__init__()
-        self.sample_rate = T(sample_rate)
-        self.buffer_size = T(buffer_size)
+        self.synthglobals = synthglobals
 
-        # Global Parameter Module
-        self.global_params = TorchSynthParameters(sample_rate, buffer_size)
```

Comment:
so this parameter is for stuff that is not within children SynthModules -- like master volume or something. Not quite the same as the new synth globals. But maybe you address this in the next PR. Will check out.

Reply:
yeah hope you like it